.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Bug Report
+about: Report a problem with qBittorrent to help us resolve it.
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Bug report
+
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+-->
+
+- [ ] I have read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good bug report with the required information
+- [ ] I have verified that **the issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php)**
+- [ ] I have **searched** the issue tracker for similar bug reports including closed ones **to avoid posting a duplicate**
```

Comment:
I very much wonder that many, even if they _will_ look through thousands of issues, would be able to come across a similar one. 
In my opinion here should (also) be  enclosed link of [frequent issues](https://github.com/qbittorrent/qBittorrent/projects/2).

Reply:
solved.